playbooks/scenario2_simple_ecs.yaml

```diff
@@ -14,7 +14,17 @@
         test_keypair_name: "{{ (prefix + '-kp') }}"
         test_network_name: "{{ ( prefix + '-net') }}"
         test_image: Standard_Fedora_33_latest
+        test_flavor: s3.medium.2
+
+    - name: Check if s3.medium.2 flavor exists
```

Comment:
please rename this task name to Query specific flavor
or something similar

Reply:
changed